app/src/components/common/__tests__/NumberInput.spec.ts

```diff
@@ -0,0 +1,36 @@
+import { shallowMount, Wrapper } from '@vue/test-utils'
+import NumberInput from '@/components/common/NumberInput.vue'
+
+describe('NumberInput.vue', () => {
+  let wrapper: Wrapper<Vue>
+
+  beforeEach(() => {
+    const label: string = 'Age'
+    const id: string = 'age'
+    const value: number = 25
+
+    wrapper = shallowMount(NumberInput, {
+      propsData: { label, id, value }
+    })
+  })
+
+  afterEach(() => {
+    wrapper.destroy()
+  })
+
+  it('renders label and input with correct props', () => {
+    expect(wrapper.find('label').text()).toBe(`${wrapper.props('label')}:`)
+    expect(wrapper.find('input').attributes('id')).toBe(wrapper.props('id'))
+    expect((wrapper.find('input').element as HTMLInputElement).value).toBe(wrapper.props('value').toString())
+  })
+
+  it('emits input event when input value changes', async () => {
+    const inputField = wrapper.find('input')
+    ;(inputField.element as HTMLInputElement).value = '30'
```

Comment:
```suggestion
    (inputField.element as HTMLInputElement).value = '30'
```

Reply:
Hi! I get an error message from the ESLint: `Unexpected newline between function and ( of function call.(no-unexpected-multiline)` when this semicolon is deleted. It seems that I have encountered a bug similar to this one: https://github.com/eslint/eslint/issues/16183 and I don't really know how to fix this issue.

Reply:
Hey, @kaytee-te and @marina-grechukha! I've tried your solution for the issue in NumberInput.spec.ts 
(`wrapper.find('input').trigger('input', {  value: '30' })`), but you were right and in such case my input does not emit the correct value. However, I fixed it by using setValue function which is equivalent to those two problematic lines of code. Here is a snippet from the documentation.
<img width="502" alt="Screenshot 2024-01-31 at 10 00 43" src="https://github.com/SolomiiaLomnytska22/solomiia-training/assets/136481621/f1e82a4a-b725-424a-a774-70e322366d0c">
Hope that this solution is fine)